### ISSUE TITLE
State Management for User Authentication

### ISSUE THREAD

**Issue by SimonNungwa** [SimonNungwa/lubenta#1]
DESCRIPTION 

Implement session-based authentication in our Express.js app using middleware like express-session for managing user login/logout states, ensuring secure and seamless user experiences.

**Comment by Nandini1071** [comment]
Hi, is this issue still open?
If yes then what is the requirement, would like to contribute.

**Comment by SimonNungwa** [comment]
@Nandini1071 Sure you can. I'll send you a txt for the requirements let me get your email

**Comment by Nandini1071** [comment]
sure email: nsabhar1071@gmail.com @SimonNungwa
Please let me know the requirements.

**Comment by Nandini1071** [comment]
Could you send me requirements on this email, would start working on this.

**Comment by Nandini1071** [comment]
And also let me know the db details how you want to test it by adding manually/dynamically data and also details please in the mail, would be helpful.

**Comment by Nandini1071** [comment]
And also let me know the db details how you want to test it by adding manually/dynamically data and also details please in the mail, would be helpful. 
@SimonNungwa

**Comment by Nandini1071** [comment]
Can you please send me the required details on the mentioned email. Thanks.
@SimonNungwa

**Comment by SimonNungwa** [comment]
@Nandini1071 sure

**Comment by Nandini1071** [comment]
Yeah thanks.

**Comment by Nandini1071** [comment]
I have some doubts also like if will do the changes in feature branch would you be merging them to your main branch, I mean is this project still in development stage ?

**Comment by SimonNungwa** [comment]
Yes this project is still in early development stage I just committed my latest changes you will receive a document in 5 minutes

**Comment by Nandini1071** [comment]
Ok would like to contribute on this and it would be helpful if in the mail you mention about the project what is it about. Thanks.

**Comment by SimonNungwa** [comment]
go ahead you can create another branch if you have any features in mind

**Comment by Nandini1071** [comment]
One doubt is this going to be live I mean for the customers or your personal project?

**Comment by SimonNungwa** [comment]
its a personal project
